app/models/post.rb

```diff
@@ -3,19 +3,16 @@ class Post < ApplicationRecord
   has_many :comments, dependent: :destroy
   has_many :likes, dependent: :destroy
 
-  validates :title, presence: true, length: { minimum: 2, maximum: 250 }
-  validates :comments_counter, presence: true, numericality: { only_integer: true, greater_than_or_equal_to: 0 }
-  validates :likes_counter, presence: true, numericality: { only_integer: true, greater_than_or_equal_to: 0 }
+  validates :title, presence: true, length: { maximum: 250 }
+  # validates :comments_counter, numericality: { only_integer: true, greater_than_or_equal_to: 0 }
+  # validates :likes_counter, numericality: { only_integer: true, greater_than_or_equal_to: 0 }
```

Comment:
- [ ] Kindly avoid leaving commented pieces of code in your codebase. It makes your codebase unclean and is not a good practice :+1: